### ISSUE TITLE
Multiple registry updates with new test/Project.toml system.

### ISSUE THREAD

**Issue by fredrikekre** [JuliaLang/Pkg.jl#1028]
Multiple registry updates with new test/Project.toml system. 
```
(v1) pkg> test Literate
  Updating registry at `~/.julia/registries/General`
  Updating git-repo `https://github.com/JuliaRegistries/General.git`
   Testing Literate
[ Info: activating environment at `/tmp/tmpAP0rYi/Project.toml`.
  Updating registry at `~/.julia/registries/General`
  Updating git-repo `https://github.com/JuliaRegistries/General.git`
 Resolving package versions...
```

**Comment by 00vareladavid** [comment]
We also probably need a better way of printing info. In other words, we print normally during the sandbox setup.

It might make sense to do something like: `Setting up `$sandboxname` sandbox' and then indent the feedback while we are setting up.

**Comment by fredrikekre** [comment]
No, I think that is fine. We printed the diff for the testing setup before too, which is nice; you want to know in what env the test are running.

**Comment by fredrikekre** [comment]
Fixed by #1033.
